helpers/response_parser.py

```diff
@@ -0,0 +1,69 @@
+"""
+Module to parse curl-like response.
+
+It is very simple implementation.
+We can upgrade it if it needed.
+"""
+from typing import Union, Optional
+
+
+class Response(object):
+
+    def __init__(
+        self,
+        http_version: str = '',
+        status: Optional[Union[str, int]] = None,
+        headers: Optional[dict] = None,
+        error: Optional[str] = None,
+    ):
+        super().__init__()
+        self.http_version = http_version
+        self.status = int(status)
+        self.headers = headers if headers else {}
+        self.error = error if error else ''
+
+    def __repr__(self):
+        return '{0} {1};\nHeaders: {2}\nErrors: {3}'.format(
+            self.http_version,
+            self.status,
+            self.headers,
+            self.error,
+        )
```

Comment:
About explicit base class: this has nothing to do with python2. It is only about consistency.
I removed 'super' - It could left from first version of PR, I already did not remember purpose of it and I fully agree It should be removed from here.
About dataclass: it is only simplified work. In the future, we will be able to want to change something(add validation or whatever). Work with class without any layer. I do not want to use dataclass here.